fellow-submissions/peter-rose.md

```diff
@@ -0,0 +1,148 @@
+# Problem Set 2.4 - Scope
+
+**1. How do variables declared with `let`, `const`, and `var` differ? Be sure to touch on _scope_, _reassignment_, and _hoisting_ for each.**
+
+<!---->
+   Firstly, const cannot change its value. Thats the purpose of it so that variable will never be able to reassign to a new value. Unlike
+   let and var which are able to be reassigned.
+   
+   Secondly, let and const hoist but you can't access them before the declaration is read by the console. Check out Questions 6 & 7
```

Comment:
> let and const hoist but you can't access them before the declaration is read by the console.

Instead... "variables defined with `let` and `const` are hoisted but can't be access until they are declared"